### ISSUE TITLE
Advanced Search: Search by Last Updated date fails

### ISSUE THREAD

**Issue by jelenadreskai** [MauroDataMapper/mdm-ui#378]
**Describe the bug**
Advanced Search: Search by Last Updated date fails. It does work for creation date.

**To Reproduce**
1. Go to Advanced Search
2. Enter search term that gives some data models in the result.
3. Enter last updated date From and To

** Actual Behavior **
Error shows up in the screen

**Expected behavior**
Data Models that match the criteria show up in the search results

**Screenshots**
![image](https://user-images.githubusercontent.com/81437684/139354102-6d6d0818-cae0-4666-963b-58f73489fd5c.png)

**Additional context**
MDM UI: 6.6.0
CODE: 4.10.0

**Comment by olliefreeman** [comment]
comment from NHSD
```
 Hello @jamesrwelch i have retested this defect and clearing of this fields is not refreshing the page compare to other search functions.

Failed Retest and can you please reopen the defect.

Thanks.
```

**Comment by olliefreeman** [comment]
@gammonpeter see the above condensed comments for the issue

**Comment by jamesrwelch** [comment]
The original issue reported is not fixed by MauroDataMapper/mdm-core#162 and is still a bug.  I believe the UI PR linked to this issue does fix the related issue reopened.
